ui/lib/open-api-explorer/addon/components/swagger-ui.js

```diff
@@ -30,13 +30,19 @@ export default class SwaggerUiComponent extends Component {
           return (
             taggedOps
               .map((tagObj) => {
-                const operations = tagObj.operations.filter((operationObj) => {
-                  return operationObj.path.includes(phrase);
+                const operations = tagObj.get('operations');
+
+                operations.filter((operationObj) => {
+                  const path = operationObj.get('path');
+                  return path.includes(phrase);
                 });
+
                 return tagObj.set('operations', operations);
               })
               // then traverse again and remove the top level item if there are no operations left after filtering
-              .filter((tagObj) => !!tagObj.operations.size)
+              .filter((tagObj) => {
+                return !!tagObj.get('operations').size;
```

Comment:
this isn't working how i expected it. currently, when i search it's not filtering things out

![image](https://github.com/hashicorp/vault/assets/903288/37bd67f9-8b4c-4d36-8fb8-5d8f88f8c462)